apps/deploy-web/src/components/deployments/DeploymentListRow.tsx

```diff
@@ -204,33 +172,37 @@ export const DeploymentListRow: React.FunctionComponent<Props> = ({ deployment,
             />
           </div>
         </TableCell>
-        <TableCell className="max-w-[100px] text-center">{deploymentName}</TableCell>
+        <TableCell className="max-w-[100px] text-center">
+          <DeploymentName deployment={deployment} deploymentServices={leaseStatus?.services} providerHostUri={provider?.hostUri} />
+        </TableCell>
         <TableCell className="text-center">
-          {isActive && isValidTimeLeft && realTimeLeft && (
-            <div className="inline-flex items-center space-x-2">
-              <span>~{formatDistanceToNow(realTimeLeft?.timeLeft)}</span>
-              {showTimeLeftWarning && (
-                <CustomTooltip
-                  title={
-                    <>
-                      Your deployment will close soon,{" "}
-                      <a href="#" onClick={showDepositModal}>
-                        Add Funds
-                      </a>{" "}
-                      to keep it running.
-                    </>
-                  }
-                >
-                  <WarningTriangle className="text-xs text-warning" />
-                </CustomTooltip>
-              )}
-            </div>
-          )}
+          <div className="flex items-center justify-center">
+            <span>{deployment.dseq || "N/A"}</span>
```

Comment:
non-blocking: this would be nice to have some margin
```suggestion
            <span className="mr-1">{deployment.dseq || "N/A"}</span>
```
current
<img width="97" alt="image" src="https://github.com/user-attachments/assets/9778ec20-b5f5-48bf-8e9e-08d9c7905198" />

with margin
<img width="129" alt="image" src="https://github.com/user-attachments/assets/c07d975a-6d6d-490d-a830-7c390d81b1e3" />